apps/back-end/src/repository/cache.repository.ts

```diff
@@ -0,0 +1,22 @@
+import Cache from 'src/model/cache.model';
+
+export const set = async (key: string, value: string, expiration?: number) => {
+    let expireAt = null;
+    if (expiration) {
+        let now = new Date();
+        expireAt = new Date(now.setSeconds(now.getSeconds() + expiration));
+    }
+
```

Comment:
問一下這邊是都設計給 token 用嗎，如果是的話感覺 expiration 可以從 token 裡面直接拿就好，就不用再從外面傳～
```suggestion
    const expirationTime = jwt.decode(value).exp;
```